packages/AutoTDD-Core.package/ATDDGitHubAction.class/instance/refreshInterval..st

```diff
@@ -0,0 +1,4 @@
+accessing
+refreshInterval: aNumber
+	
+	refreshInterval := aNumber
```

Comment:
Not sure if this suggested change is very squeak-like, but it may be a good idea to assert that we are passing a valid number to the refresh interval?

Reply:
This is something that our GUI should handle. The GUI should make sure the User Input is fine and the Core can therefore always assume that all inputs are legit. As a matter of fact, check the file: `AutoTDD-GUI` > `ATDDGitHubActionEditor` > `save` which calls `hasValidRefreshInterval`. This function makes sure the entered interval is actually a number. So the validation is already implemented :)